openfe/tests/protocols/test_openmmutils.py

```diff
@@ -275,7 +275,7 @@ def test_free_energies(self, analyzer):
             ret_dict['forward_and_reverse_energies']['forward_dDGs'].m,
             np.array([0.077645, 0.054695, 0.044680, 0.03947, 0.034822,
                       0.033443, 0.030793, 0.028777, 0.026683, 0.026199]),
-            rtol=1e-01,
+            rtol=5e-01,
```

Comment:
can we do either `rtol=2e-01` or `atol=0.01`?

Reply:
So the tests are failing on a difference of 0.083136 to 0.088335. This makes for a relative tolerance of just shy of 5e-01.

We could go for atol of 0.01 but that's a lot looser a check than rtol of 5e-01.

Reply:
I am happy with either -- pick one and I will change it :)

Reply:
let's go with `rtol=5e-01`